modules/enabling_encapsulation.adoc

```diff
@@ -0,0 +1,143 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/optimizing-cpu-usage.adoc
+
+[id="enabling-encapsulation_{context}"]
+= Configuring mount namespace encapsulation
+
+You can configure mount namespace encapsulation so that the cluster runs with less resource overhead.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the cluster-admin role.
+
+* You have installed the OpenShift CLI (oc).
+
+.Procedure
+
+. Create a file called `mount_namespace_config.yaml` with the following YAML to control whether encapsulation is enabled or disabled:
+
++
+[source,yaml]
+----
+  apiVersion: machineconfiguration.openshift.io/v1
+  kind: MachineConfig
+  metadata:
+    labels:
+      machineconfiguration.openshift.io/role: master
+    name: 99-kubens-master
+  spec:
+    config:
+      ignition:
+        version: 3.2.0
+      systemd:
+        units:
+        - enabled: true <1>
+          name: kubens.service
+  ---
+  apiVersion: machineconfiguration.openshift.io/v1
+  kind: MachineConfig
+  metadata:
+    labels:
+      machineconfiguration.openshift.io/role: worker
+    name: 99-kubens-worker
+  spec:
+    config:
+      ignition:
+        version: 3.2.0
+      systemd:
+        units:
+        - enabled: true <2>
+          name: kubens.service
+----
+<1> If `enabled` is set to true, encapsulation is in effect for all control plane (previously known as "master") nodes.
+<2> If `enabled` is set to false, encapsulation is disabled for all worker nodes.
+
+. Apply the mount namespace `MachineConfig` CR:
++
+[source,terminal]
+----
+$ oc apply -f mount_namespace_config.yaml
+----
+
+.Example output
+[source,terminal]
+----
+machineconfig.machineconfiguration.openshift.io/99-kubens-master created
+machineconfig.machineconfiguration.openshift.io/99-kubens-worker created
+----
+. The `MachineConfig` CR can take up to 30 minutes to finish being applied in the cluster. You can check the status of the `MachineConfig` CR by running the following command:
+[source,terminal]
+----
+$ oc get mcp
+----
++
+.Example output
+[source,terminal]
+----
+NAME     CONFIG                                             UPDATED   UPDATING   DEGRADED   MACHINECOUNT   READYMACHINECOUNT   UPDATEDMACHINECOUNT   DEGRADEDMACHINECOUNT   AGE
+master   rendered-master-03d4bc4befb0f4ed3566a2c8f7636751   False     True       False      3              0                   0                     0                      45m
+worker   rendered-worker-10577f6ab0117ed1825f8af2ac687ddf   False     True       False      3              1                   1                     0                      45m
+----
+Prior to verifying whether the feature has been successfully enabled/disabled, we must wait for `MachineConfig` to be applied successfully across all master and worker nodes. This can be done as follows:
+[source,terminal]
+----
+$ oc wait --for=condition=Updated mcp --all --timeout=30m
+----
++
```

Comment:
I see a lot of these odd `+` characters in the document now, and I think they're interfering with the output.

Reply:
Yes. We're wrestling with inconsistencies in AsciiDoctor rendering. WIP.

Reply:
All Example output blocks working correctly.